contentcuration/contentcuration/frontend/administration/pages/Users/UserActionsDropdown.vue

```diff
@@ -36,7 +36,7 @@
       v-model="emailDialog"
       :query="{ ids: [userId] }"
     />
-    <VMenu offsetY>
+    <Menu>
```

Comment:
I can see in the notes that you updated `<Menu />` to use the lazy setting (and can see the prop added there). Is `lazy` not an option for `VMenu` which is why you switched just to `<Menu/>`? Or is this for another reason?

Reply:
`lazy` is indeed an option for `VMenu`. Our `Menu` inherits `VMenu`, and so I updated this since it also defaults the `offsetY` attribute-- I'm sure we'll want that 99% of the time for our menus anyways. There were many usages of `Menu`, so by adding `lazy` there, I took advantage of that abstraction layer requiring a single change. We can do the same when we migrate off Vuetify and to KDS for things like this.